public/bitemp_graph_helper.js

```diff
@@ -541,6 +541,9 @@ var getBarChart = function (params, docProp) {
 
   $('#deleteButton').click(function() {
     setupDelete(chart);
+    var date = moment().toISOString();
+    date = date.split('.');
+    $("#sysStartBox").val(date[0]);
```

Comment:
Ok Hil, since you're populating the sysBox with the current date, check out deleteSuccess function and see if there is anything you can remove/shorten...